PENDING.md

```diff
@@ -38,6 +38,7 @@ BREAKING CHANGES
 * [types] sdk.NewCoin now takes sdk.Int, sdk.NewInt64Coin takes int64
 * [cli] #1551: Officially removed `--name` from CLI commands
 * [cli] Genesis/key creation (`init`) now supports user-provided key passwords
+* [cli] `gaiacli advanced` no longer exists - to access `ibc`, `rest-server`, and `validator-set` commands use `gaiacli ibc`, `gaiacli rest-server`, and `gaiacli tendermint`, respectively
```

Comment:
Can you reference the PR here? e.g. `[cli] #2014 ... ` or even reference the actual PR? e.g. `[cli] [#2014](link) ...`